src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
I don't see a need for inheritance here, `Lottery` is not a `ColorSupplier`. And it's not using or overriding any methods from a parent class